### PR TITLE
feat: new method to get commit date from git repo

### PR DESCRIPTION
**Pull request by trim21**
for https://github.com/docker/metadata-action/pull/308

### CODE REVIEW

File: src/buildkit/git.ts [hunk 1]
```diff
@@ -110,4 +113,28 @@ export class Git {
     res.shortName = repoSplitBySlash[repoSplitBySlash.length - 1].replace('.git', '');
     return res;
   }
+
+  public static async commitDate(ref: string, gitDir: string | undefined = undefined): Promise<Date> {
```
Comment (crazy-max) [inline src/buildkit/git.ts:117]: Should be implemented in https://github.com/docker/actions-toolkit/blob/main/src/git.ts, not the buildkit one.

**Review by crazy-max — CHANGES_REQUESTED** [review]

File: src/buildkit/git.ts (hunk 1 above)
Comment (trim21) [reply src/buildkit/git.ts]: done

**Comment by trim21** [comment]
is it ok now?

**Comment by trim21** [comment]
looks like ci failed with other tests.

**Pull request closed without merging**